src/main/java/org/apache/sling/dynamicinclude/Configuration.java

```diff
@@ -97,6 +98,11 @@
 
   static final String PROPERTY_REWRITE_PATH = "include-filter.config.rewrite";
 
+  static final String NAME_HINT_PROPERTY_NAME = "webconsole.configurationFactory.nameHint";
```

Comment:
Can you make this private?

Reply:
Ah, now I see why they're all package-private :)

Notice how the properties are defined outside the class.

```
@Component(metatype = true, configurationFactory = true, label = "Apache Sling Dynamic Include - Configuration", immediate = true, policy = ConfigurationPolicy.REQUIRE)
@Service(Configuration.class)
@Properties({
// private members of Configuration cannot be accessed here.
})
public class Configuration {
   // class body
}
```
If we keep the properties defined the way they are, the constants need to be visible outside the class (so package-private is the next most restrictive scope).

Alternatively, I could refactor the whole thing so that the property definitions sit inside the `Configuration` class. Or extract the names and default values to a separate class. Still, that class would need to be package-private at best so no real gain here.

Reply:
Per http://felix.apache.org/documentation/subprojects/apache-felix-web-console.html the property for nameHint should not be private. However I’m not sure if that is still applicable with the newer annotations

Reply:
@JEBailey this class still uses the SCR annotations. A rewrite should be fairly straightforward but that sounds like a relatively major version change.

Reply:
@toniedzwiedz - I think we can keep them as they are, not worth the fuss. Does this mean that the version bump needs to stay as well?